coffee/lib/abstract-chosen.coffee

```diff
@@ -280,8 +281,17 @@ class AbstractChosen
     if /Android/i.test(window.navigator.userAgent)
       return false if /Mobile/i.test(window.navigator.userAgent)
     return true
+    
+  @tagname_is_valid: : (@form_field) ->
```

Comment:
yeah : is a typo
I correct identation (the problems were wrong identations)
I successfully compiled it with npm coffee 1.9.0